server/src/core/stat/tests/fixtures.ts

```diff
@@ -0,0 +1,83 @@
+export const transactions = [
+  {
+    _id: 'transaction1',
+    expectedAmount: 250,
+    to: "id 1",
+    from:"",
+    type: "donation",
+    status: "paymentRequested",
+    amount: 0,
+  },
+  {
+    _id: 'transaction2',
+    expectedAmount: 470,
+    to: "id 1",
+    from:"",
+    type: "donation",
+    status: "success",
+    amount: 470,
+  },
```

Comment:
You could create another successful donation to the same id just to ensure that the "totalContributors" figure does not count the same donor twice even if they have multiple transactions.